src/commands/projects.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * This project follows the example provided in the Yards documentation for command hierarchy and directory structure. 
+ * @see: https://github.com/yargs/yargs/blob/master/docs/advanced.md#commanddirdirectory-opts
+ */
+exports.command = 'projects <command>'
+exports.desc = 'Manage GitHub projects'
```

Comment:
```suggestion
exports.desc = 'Manage GitHub project boards'
```

https://help.github.com/en/articles/about-project-boards